src/Web/UIEvent/InputEvent.purs

```diff
@@ -23,3 +26,9 @@ toEvent = unsafeCoerce
 foreign import data_ :: InputEvent -> String
 
 foreign import isComposing :: InputEvent -> Boolean
+
+foreign import inputType_ :: InputEvent -> String
```

Comment:
I'd suggest putting the `_` at the start rather than the end (that's the usual convention for private FFI functions), and give the module an export list explicit with this hidden from the exports.

The `_` at the end for `data` is because `data` is a reserved syntactic term in PS. :wink: